core/Symbols.h

```diff
@@ -87,6 +87,7 @@ class Symbol final {
         static constexpr u4 METHOD_FINAL = 0x0000'0800;
         static constexpr u4 METHOD_OVERRIDE = 0x0000'1000;
         static constexpr u4 METHOD_IMPLEMENTATION = 0x0000'2000;
+        static constexpr u4 METHOD_UNSAFE_IMPL = 0x0000'4000;
```

Comment:
WDYT about METHOD_INCOMPATIBLE_OVERRIDE?
Unsafe is too general.

Reply:
I've changed this to `incompatible_override`. It's no longer needed for RBI compatibility, but I've used it in the corresponding monorepo PR.